src/Glue.elm

```diff
@@ -1,24 +1,24 @@
 module Glue exposing
     ( Glue
-    , glue, simple, poly
+    , simple, poly, glue
     , init
-    , update, updateModel, updateWithTrigger, updateModelWith, trigger
+    , update, updateModel
```

Comment:
* `updateWithTrigger` is the same as `update`
* we talked about `trigger` - unseen in the wild, unsure if useful

Reply:
the difference is that `updateWithTrigger` don't expect an extra argument of `msg`.  That is important as it's legit to have such function (without extra argument before model). So in theory `updateWithTrigger` can replace `update` but not the other way around.

The good thing about `update` is that it allows you to simply define partialy applied function without `<<`.

```elm
let updateFoo = Glue.update foo Foo.update
in case msg of
    FooMsg m ->
        updateFoo m (model, Cmd.none)
```

Now the question is if it's worth it. Maybe it's not. Given all this - do you still think it's better to remove those?

Reply:
oh I missed it. I'll put it back then. Do you think `updateModel` should have one like this too?

Reply:
well.... I think you still might be right... you can simulate this:

```elm
let updateFoo m = Glue.update foo (Foo.update m)
in case msg of
    FooMsg m ->
        updateFoo m (model, Cmd.none)
```

The only important part would be to keep API of `updateWithTrigger` and not `update`. It would make it a bit harder to understand in most common `update` function but I'm not 100% sure if that's a big deal. Maybe exposing less is better. Hard design choices :/

Reply:
btw the same applies to `updateModel` and `updateModelWith` there are analogs (without Cmds)